kuksa_feeders/gps2val/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Copyright Robert Bosch GmbH, 2020. Part of the Eclipse Kuksa Project.
+#
+# All rights reserved. This configuration file is provided to you under the
+# terms and conditions of the Eclipse Distribution License v1.0 which
+# accompanies this distribution, and is available at
+# http://www.eclipse.org/org/documents/edl-v10.php
+#
+
+FROM python:3.8-alpine
+RUN apk update && apk add gpsd
```

Comment:
PLease make two stage build, or at least do not cache everything.

This dockerfile is quite unoptimised (and space on our Jenkins Masters is limted)

Reply:
I am not sure if it is possible to optimise. because I need gpsd for runtime. Do you know, how to tell apk a location to install a software?

Reply:
At least use `apk --no-cache`

Reply:
2MB smaller